cmd/blockchaincmd/import_public.go

```diff
@@ -69,6 +70,8 @@ flag.`,
 		"",
 		"the blockchain ID",
 	)
+	cmd.Flags().StringVar(&rpcURL, "rpc", "", "rpc endpoint for the blockchain")
```

Comment:
whats the difference between nodeEndpoint and rpcURL? I assume for most users, rpcURL = nodeEndpoint.
You can get the same information from nodeEndpoint as rpcURL (blockchainID, vmType, etc). Maybe except for VM Version? 

IMO we should just keep 1 URL flag whether its nodeEndpoint or RPCURL to prevent user confusion

Reply:
Node endpoint is basically the ip of a public tracking node, rpc endpoint is the url provided by an API node.
The first one is used to directly access Pchain on the L1 validator so as to get the vm version.
The second one is how to access the L1.
Do you propose to remove all vm version stuff altogether? This is outside of this PR but can be done anyway.
which should be your proposal to have a vm version set in sidecar, as a replacement of current code?

Reply:
I think we can just merge rpcURL and node Endpoint flags together. I think we can just keep rpc url and remove node endpoint. Basically we automatically detect if the url provided is RPC or end point. If it is node endpoint then we get the vm info from that url

Reply:
we may need both usually, so most probably we remove node endpoint

Reply:
if you agree we can just remove the node endpoint stuff @sukantoraymond

Reply:
ya we can remove node endpoint